platform-sdk/consensus-otter-tests/build.gradle.kts

```
@@ -46,8 +47,10 @@ tasks.register<Test>("testTurtle") {
     jvmArgs("-XX:ActiveProcessorCount=6")
 }
 
-// Runs tests against the Solo environment
-tasks.register<Test>("testSolo") {
+// Runs tests against the Container environment
+tasks.register<Test>("testContainer") {
+    dependsOn(":consensus-otter-docker-app:assemble")
```

```suggestion
    dependsOn(":consensus-otter-docker-app:copyDockerizedApp")
```

...to be clear on what exactly is needed.

The direct `dependsOn` is always an indication that it could be somehow "better modeled" with dependencies/requires. But as this is not a Jar, but a folder that is filled, I think it is a fine pragmatic solution. Maybe something to revisit if the project structures evolve further.